### ISSUE TITLE
-U update switch upgrades to youtube-dl instead of youtube-dlc

### ISSUE THREAD

**Issue by l00tzOMG** [blackjack4494/youtube-dlc#124]
When I tried updating the program with the `-U` switch it downloads the original youtube-dl instead of the modded version.

**Comment by blackjack4494** [comment]
Yes this is still not fixed. I am aware of this problem. Will be done in next release ;)

**Comment by jyrkive** [comment]
`-U` isn't managing to update youtube-dlc at all at the moment. Trying it with version 2020.09.23-2 (second latest as of writing this) gives this error message:

> ERROR: can't find the current version. Please try again later.
